Formula/d/daemontools.rb

```diff
@@ -35,6 +35,7 @@ def install
     cd "daemontools-#{version}" do
       inreplace ["package/run", "src/svscanboot.sh"] do |s|
         s.gsub! "/service", "#{etc}/service"
+        s.gsub! "/command", "#{bin}"
```

Comment:
```suggestion
        s.gsub! "/command", bin.to_s
```

Reply:
Thanks, I appreciate it! Sincerely. Commit pushed.

Reply:
Sorry just noticed the commit message issue. I think I need to squash and force push so that's incoming.

Reply:
Thanks! Now that we know it builds and tests out successfully with this change, would you mind trying to build it from source on your machine and ensure it works as you expect? That will be `HOMEBREW_NO_INSTALL_FROM_API=1 brew install -sv daemontools`. If it's already installed, just change `install` to `reinstall`. If you can confirm it does what you expect, I'll get this merged.

Reply:
Confirmed - this works!

```console
➜ pwd
/opt/homebrew/etc/service/example

➜ cat run
#!/bin/sh

counter=1

while true; do
  echo $counter
  counter=$((counter+1))
  sleep 1
done

➜ ./run
1
2
3

➜ brew services start daemontools
Warning: daemontools must be run as root to start at system startup!
==> Successfully started `daemontools` (label: homebrew.mxcl.daemontools)

➜ svstat .
.: up (pid 34304) 38 seconds

➜ ps aux | grep 34304
erichdblume      34594   0.0  0.0 408626896   1392 s004  S+    3:52PM   0:00.00 grep 34304
erichdblume      34304   0.0  0.0 408507888   2336   ??  S     3:52PM   0:00.05 /bin/sh ./run

➜ kill -9 34304

➜ svstat .
.: up (pid 34663) 3 seconds
```

Reply:
@p-linnane , I deeply appreciate your help here. I think this PR is ready for you to take over now.

Sorry for the bad behavior I displayed on Thursday, sorry for my snark, and also on re-reading it I just wanted to add that mentioning Nix was a mistake. It was meant as a joke about how I was running in to effectively the same problem there as I had with homebrew (namely, getting the correct service config file to be loaded) after being told homebrew had no support to offer. But on re-reading I see how it came across, well, very poorly. That and all the rest. Anyway, best wishes. Thanks for the support.

Reply:
I appreciate your apology. It's exceedingly rare in the world of open source, and why I linked that article in our other discussion. I think this has gone in the best way possible. Your issue is fixed, you are now a contributor to Homebrew, and we're ending this interaction on positive terms. I hope we can look forward to more contributions from you in the future.